### ISSUE TITLE
location.hostname is supported on IE11

### ISSUE THREAD

**Issue by targumon** [mdn/browser-compat-data#4336]
As opposed to the compat table under [Location](https://developer.mozilla.org/en-US/docs/Web/API/Location), `location.hostname` is actually supported on IE11.

I'd gladly make a PR, but since I'm only able to validate IE11 (and not previous versions) so I'm not sure if to just flip the value to `true`, or set it to `11`.

**Comment by targumon** [comment]
Update: found an IE8 (on Windows XP...) and it works there as well. In fact, with this result:
```
>> for (k in location) console.log(k)
LOG: hostname
LOG: href
LOG: host
LOG: hash
LOG: port
LOG: pathname
LOG: search
LOG: protocol
```
I have no reason to believe `hostname` wasn't supported as long as the rest
([caniuse.com](https://www.caniuse.com/#search=window.location) seems to agree)

Going to make a PR.

**Comment by vinyldarkscratch** [comment]
Resolved by #4355.
